src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -25,12 +25,29 @@
 	<div class="contact-header-avatar">
 		<div class="contact-header-avatar__wrapper">
 			<div class="contact-header-avatar__background" @click="toggleModal" />
+
 			<div v-if="contact.photo"
+				v-click-outside="closeMenu"
 				:style="{ 'backgroundImage': `url(${contact.photoUrl})` }"
-				class="contact-header-avatar__photo"
-				@click="toggleModal" />
+				class="avatar contact-header-avatar__options contact-avatar-options contact-header-avatar__photo"
+				@click="toggleModal">
+				<input id="contact-avatar-upload"
+					ref="uploadInput"
+					type="file"
+					class="hidden"
+					accept="image/*"
+					@change="processFile">
+			</div>
+
+			<div v-if="contact.photo"
+				class="overlay"
```

Comment:
Please use the BEM css class naming methods :)
```suggestion
				class="contact-header-avatar__overlay"
```

Reply:
OK, I can do that.